firmware/controllers/algo/launch_control.cpp

```diff
@@ -78,6 +70,7 @@ bool LaunchControlBase::isInsideSwitchCondition() const {
 		
 	default:
 		// ALWAYS_ACTIVE_LAUNCH
+		engineConfiguration->launchDisableBySpeed = 1;
```

Comment:
changing engineConfiguration at runtime is a bit unexpected

maybe that's NOT _configuration_ but runtime state? please move this field to "Engine" class?

Reply:
Yes, changing config at runtime is always wrong (unless it's some "set config x" command)

Reply:
it is not, i moved inside launchControl class, since we use it as a simple variable.